dotcom-rendering/docs/contributing/detailed-setup-guide.md

```diff
@@ -89,16 +89,13 @@ You may want local identity cookies to be available in `dotcom-rendering`. To en
 
 -   `make build` creates production-ready bundles.
 -   `make start` starts the production server.
--   `make stop` stops the production server.
 
 You may need to run these with `sudo`
 
-Production environment uses `pm2`. More scripts can be found in the `makefile` [scripts](https://github.com/guardian/dotcom-rendering/blob/e2c020f7e0ed24751ea729eec93f1271d37e3b50/makefile#L31)
```

Comment:
For future travellers, is `pm2` no longer useful? I'd be wary of making changes to PROD as part of this PR, but maybe this one is small enough to sneak in?

Reply:
I was a little too keen removing this. I see we are still using pm2